tests/manual/testserviceworkers.html

```diff
@@ -0,0 +1,32 @@
+<html>
+    <body>
+        <h2>ServiceWorker Test</h2>
+        <p id="p1"></p>
+        <script>
+            function log(txt) {
+                document.getElementById("p1").innerHTML += txt + "<br>";
+            }
+
+
+            if (navigator.serviceWorker) {
+                    navigator.serviceWorker.onmessage = function(event) {
```

Comment:
Wrong indentation. You should drop 4 spaces.

Reply:
oh. indeed - missed this one...